client/public/runner.html

```diff
@@ -54,6 +54,28 @@
       window.console = consoleProxy;
       window.addEventListener("error", (e) => console.log(e.error));
 
+      async function loadExternalScripts(html) {
+        const dummy = document.createElement("div");
+        dummy.innerHTML = html;
+        const scripts = dummy.querySelectorAll("script[src]");
+
+        const loaded = [...scripts].reduce((acc, scrpt) => {
+          acc.push(
+            new Promise((resolve) => {
+              const script = document.createElement("script");
+              script.src = scrpt.src;
+              script.addEventListener("load", () => resolve());
+
+              document.head.appendChild(script);
+            })
+          );
+
+          return acc;
+        }, []);
```

Comment:
I think we can just `.map()`?

```suggestion
        const loaded = [...scripts].map((scrpt) => new Promise((resolve) => {
          const script = document.createElement("script");
          script.src = scrpt.src;
          script.addEventListener("load", () => resolve());

          document.head.appendChild(script);
        }));
```

Reply:
Oh, right. It was left from my previous approach and I missed to changed it. Thanks!